thriftbp/prometheus.go

```diff
@@ -88,3 +88,93 @@ var (
 		Help: "The number of in-flight requests",
 	}, clientActiveRequestsLabels)
 )
+
+var (
+	serverConnectionsGauge = promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "thrift_server_connections",
+		Help: "The number of client connections established to the service",
+	})
+)
+
+var (
+	ttlClientReplaceLabels = []string{
+		serverSlugLabel,
+		successLabel,
+	}
+
+	ttlClientReplaceCounter = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "thrift_ttl_client_connection_houskeeping_total",
+		Help: "Total connection housekeeping done in thrift ttlClient",
```

Comment:
"successLabel" doesn't make sense to me in this context; what is a successful vs a failed "housekeeping"?

Reply:
whether we successfully replaced the connection in the background.

Reply:
🔕 so maybe a name like `thriftbp_ttlclient_connection_recycle_total` and a name like `replacement_success`?

or maybe just update the doc a little?

Reply:
updated the doc.